revscores/languages/language.py

```diff
@@ -0,0 +1,19 @@
+
+class Language:
+    """
+    Constructs a Language instance that wraps two functions, "is_badword" and "is_misspelled" -- which will
+    check if a word is "bad" or does not appear in the dictionary.
+    """
+    def __init__(self, is_badword, is_misspelled):
+        self.is_badword = is_badword
+        self.is_misspelled = is_misspelled
+    
+    def badwords(self, words):
```

Comment:
Do we need to document that this takes a list of words and returns the sublist of those which are bad? (same question below, about misspellings)